### PR TITLE
Add an example of clone feature

### PR DESCRIPTION
**Pull request by erthalion**
Since this feature is quite important it makes sense to mention it in the user part.

### CODE REVIEW

File: docs/user.md [hunk 1]
```diff
@@ -181,3 +181,40 @@ spec:
...
+metadata:
+  name: acid-test-cluster
+spec:
+  clone:
+    uid: "efd12e58-5786-11e8-b5a7-06148230260c"
+    cluster: "acid-batman"
+    timestamp: "2017-12-19T12:40:33+01:00"
+```
+
+Here `cluster` is a name of a target cluster that is going to be cloned. If
```
Comment (alexeyklyukin) [inline docs/user.md]: `cluster` is the name of the cluster to be cloned.

File: docs/user.md [hunk 2]
```diff
@@ -181,3 +181,40 @@ spec:
...
+`timestamp` is not empty, then a new cluster will be cloned from an S3 bucket,
```
Comment (alexeyklyukin) [inline docs/user.md]: the operator does not create S3 buckets. Perhaps "a new cluster will be cloned from S3 ... "

File: docs/user.md [hunk 3]
```diff
@@ -181,3 +181,40 @@ spec:
...
+that was created by operator,
+using the latest backup before the `timestamp`. In this case `uid` field is
```
Comment (alexeyklyukin) [inline docs/user.md]: In this case,

File: docs/user.md [hunk 4]
```diff
@@ -181,3 +181,40 @@ spec:
...
+also mandatory - operator will use it to find an S3 bucket. You can find this
```
Comment (alexeyklyukin) [inline docs/user.md]: the correct folder inside the S3 bucket.

File: docs/user.md [hunk 5]
```diff
@@ -181,3 +181,40 @@ spec:
...
+field from metadata of a target cluster:
```
Comment (alexeyklyukin) [inline docs/user.md]: target is the one to be cloned to. I'd say it is the source (or origin) cluster.

File: docs/user.md [hunk 6]
```diff
@@ -181,3 +181,40 @@ spec:
...
+
+```yaml
+apiVersion: acid.zalan.do/v1
+kind: postgresql
+metadata:
+  name: acid-test-cluster
+  uid: efd12e58-5786-11e8-b5a7-06148230260c
+```
+
+If `timestamp` is empty or absent, a new cluster will be cloned from an
```
Comment (alexeyklyukin) [inline docs/user.md]: the cluster to clone is assumed  to be running and the clone procedure invokes pg_basebackup from it. Note that the timestamp parameter mandates the timezone.

File: docs/user.md (hunk 4 above)
Comment (zerg-junior) [reply docs/user.md]: correct key maybe ? afaik there are no folders within S3 buckets, keys just mimic directory structure using names with slashes.

Comment (alexeyklyukin) [reply docs/user.md]: although  directory (or folder) is universally understood in the context of S3 (I think WAL-E mentions it at least once on their page), using key would things less ambiguous, so +1

**Comment by alexeyklyukin** [comment]
👍

**Comment by erthalion** [comment]
:+1:

**Pull request merged**